store/sqlstore/team_store.go

```diff
@@ -669,9 +669,8 @@ func (s SqlTeamStore) GetTotalMemberCount(teamId string) store.StoreChannel {
 	})
 }
 
-func (s SqlTeamStore) GetActiveMemberCount(teamId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		count, err := s.GetReplica().SelectInt(`
+func (s SqlTeamStore) GetActiveMemberCount(teamId string) (int64, *model.AppError) {
+	count, err := s.GetReplica().SelectInt(`
 			SELECT
```

Comment:
nit: Probably 1 less tab for each line of the query string would make it looks better, and probably, moving the entire query to a query variable can be good too.

Reply:
Are you sure about a new query var is needed here ? The others methods follow this pattern. 
As to the tab for each line I think is ok